src/extension.ts

```diff
@@ -8,6 +8,7 @@ import CompletionProvider from './providers/completion';
 import SymbolsProvider from './providers/symbols';
 import DocumentDecorator from './todo/decorators/document';
 import ChangesDecorator from './todo/decorators/changes';
+import ChangesCommands from './changes';
```

Comment:
You're importing "changes" but you're calling it "ChangesCommands", I guess either the variable name or the filename should be renamed.